zcash_client_backend/Cargo.toml

```diff
@@ -23,7 +23,7 @@ development = ["zcash_proofs"]
 zcash_address = { version = "0.2", path = "../components/zcash_address" }
 zcash_encoding = { version = "0.2", path = "../components/zcash_encoding" }
 zcash_note_encryption = { version = "0.2", path = "../components/zcash_note_encryption" }
-zcash_primitives = { version = "0.10", path = "../zcash_primitives" }
+zcash_primitives = { version = "0.10", path = "../zcash_primitives", default-features = false }
```

Comment:
This change should be documented in the changelog. Almost all downstream users likely don't have to do anything as they will already have a dependency on `zcash_primitives` that doesn't disable default features (because `default-features = false` previously didn't do anything for it), but anyone who added it by accident or by force of habit, or who doesn't have a direct dependency on `zcash_primitives` (unlikely as we don't re-export types) will see a performance regression.